.ci/checker-framework.groovy

```diff
@@ -7,8 +7,8 @@ import groovy.xml.XmlUtil
 import java.util.regex.Matcher
 import java.util.regex.Pattern
 
-@Field static final String USAGE_STRING = "Usage groovy " +
-        ".${File.separator}.ci${File.separator}checker-framework.groovy" +
+@Field final String USAGE_STRING = "Usage groovy " +
```

Comment:
Keep static, it not related to your issue

Reply:
Okay!